test-api-generator/citrus-openapi-codegen/src/main/java/org/citrusframework/openapi/generator/CitrusJavaCodegen.java

```diff
@@ -474,6 +492,10 @@ private CustomCodegenParameter convertToCustomCodegenParameter(
         return customCodegenParameter;
     }
 
+    private boolean isGenerateModels() {
```

Comment:
```suggestion
    private boolean isModelGenerationEnabled() {
```

maybe rethink this name 😉